MaxText/maxengine_server.py

```diff
@@ -56,6 +56,7 @@ def main(config):
       enable_jax_profiler=config.enable_jax_profiler if config.enable_jax_profiler else False,
       jax_profiler_port=config.jax_profiler_port if config.jax_profiler_port else 9999,
       enable_model_warmup=config.enable_model_warmup if config.enable_model_warmup else False,
+      multi_sampling=config.multi_sampling if config.multi_sampling else False,
```

Comment:
What will the the error if just passes `config.multi_sampling`? I am not sure why other above params in this format.

Reply:
Can `config.multi_sampling` be None?

Reply:
Why is that? by default `multi_sampling: False`?